### PR TITLE
bug 1537857. Fix retrieving prometheus metrics

### PR DESCRIPTION
**Pull request by jcantrill**
This PR:

* bumps openshift-elasticsearch-plugin to fix retrieving metrics
* bumps the prometheus exporter to fix issue related to SG and plugin

Depends on: 
* https://github.com/fabric8io/openshift-elasticsearch-plugin/pull/119
* https://github.com/fvvanholl/elasticsearch-prometheus-exporter/pull/82
* https://github.com/openshift/openshift-ansible/pull/6903

### CODE REVIEW

File: elasticsearch/Dockerfile.centos7 [hunk 1]
```diff
@@ -26,7 +26,7 @@ ENV ES_CLOUD_K8S_VER=2.4.4_01 \
 ARG ES_CLOUD_K8S_VER=2.4.4_01
 ARG OSE_ES_VER=2.4.4.20
 ARG ES_CLOUD_K8S_URL
-ARG OSE_ES_URL
+ARG OSE_ES_URL=https://github.com/jcantrill/openshift-elasticsearch-plugin/releases/download/2.4.4.21-snapshot/openshift-elasticsearch-plugin-2.4.4.21-SNAPSHOT.zip
```
Comment (ewolinetz) [inline elasticsearch/Dockerfile.centos7]: this will change once the plugin is released, correct?

Comment (jcantrill) [reply elasticsearch/Dockerfile.centos7]: yes.  Thus the reason i made it do not merge

**Comment by richm** [comment]
I suggest added a test for the prometheus authentication to the access_control.sh test - make sure that the prometheus user has correct access, and make sure no other user has access to prometheus.

**Comment by ewolinetz** [comment]
approving based on discussion with @jcantrill on irc:

```
ewolinetz, the changes resolve the issue... but im wokring on a bdd based test.. not bash                                           
but i dont think it will be in by wed                                                                                              
locally my bdd test confirms the fix 
```

/lgtm

**Pull request merged**